client/src/App.js

```diff
@@ -1,13 +1,31 @@
-import './App.css';
-import ShowAllProjects from './components/ShowAllProjects';
-import allProjects from './db';
+import './App.css'
+// import ShowAllProjects from './components/showAllProjects/ShowAllProjects'
+import allProjects from './db'
+import { useState, useEffect } from 'react'
+import AddProject from './pages/AddProject'
+import AllProjects from './pages/AllProjrcts'
+import { Routes, Route } from 'react-router-dom'
+
 
 
 function App() {
+    const [newPr, setNewPr] = useState(allProjects)
 
+    function NewProjectToDb({event, ID, project_name, client, status, employee_name, work_hours, start_date, end_date, sum_hours}){
+        const newProjectLi = [...newPr, {ID, project_name, client, status, employee_name, work_hours, start_date, end_date, sum_hours, in_detail: false }]
```

Comment:
Same here as written above: It is better not to use abbreviations for better understanding your code. Therefore, I would rename `newProjectLi` to `newProjectList`.